### DISCUSSION TITLE
PageFind Filtering

### DISCUSSION THREAD

**Discussion by Karlstens** [ATQQ/sugar-blog#244] [Q&A] [answered]
Does PageFind filtering work with vitepress-plugin-pagefind? Would anyone have an example site showing this feature?

https://pagefind.app/docs/filtering/

**Comment by ATQQ** [comment, marked as answer]
Can you see if this built-in filter can help you?

https://github.com/ATQQ/sugar-blog/tree/master/packages/vitepress-plugin-pagefind#42-search-result-filter

**Reply by Karlstens** [reply]
I was more curious about the filtering feature and how other developers may have implemented PageFind filtering in their sites/search. 

My current project doesn't have a specific need for filtering, rather - I'm just looking to learn about the feature for gaining an understanding of its possible/potential future use.

**Reply by ATQQ** [reply]
Ok, I didn’t delve deeply into the usage of pagefind’s official filter. I just added the filter configuration item in the same way as regular search filtering, so that the user can simply operate the results to be displayed.

The following is the specific code location

https://github.com/ATQQ/sugar-blog/blob/c2c089b9647de5155e512c0c4a235b6aa0605a51/packages/vitepress-plugin-pagefind/src/Search.vue#L156-L158
